_templates/page.html

```diff
@@ -0,0 +1,146 @@
+{% extends "base.html" %}
+
+{% block body -%}
+{% include "partials/icons.html" %}
+
+<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation">
+<input type="checkbox" class="sidebar-toggle" name="__toc" id="__toc">
+<label class="overlay sidebar-overlay" for="__navigation"></label>
+<label class="overlay toc-overlay" for="__toc"></label>
+
+{% if theme_announcement -%}
+<div class="announcement">
+  <aside class="announcement-content">
+    {% block announcement %} {{ theme_announcement }} {% endblock announcement %}
+  </aside>
+</div>
+{%- endif %}
+
+<div class="page">
+  <header class="mobile-header">
+    <div class="header-left">
+      <label class="nav-overlay-icon" for="__navigation">
+        <i class="icon"><svg><use href="#svg-menu"></use></svg></i>
+      </label>
+    </div>
+    <div class="header-center">
+      <a href="{{ pathto(master_doc) }}"><div class="brand">{{ docstitle }}</div></a>
+    </div>
+    <div class="header-right">
+      <label class="toc-overlay-icon toc-header-icon{% if furo_hide_toc %} no-toc{% endif %}" for="__toc">
+        <i class="icon"><svg><use href="#svg-toc"></use></svg></i>
+      </label>
+    </div>
+  </header>
+  <aside class="sidebar-drawer">
+    <div class="sidebar-container">
+      {% block left_sidebar %}
+      <div class="sidebar-sticky">
+        {%- for sidebar_section in sidebars %}
+          {%- include sidebar_section %}
+        {%- endfor %}
+      </div>
+      {% endblock left_sidebar %}
+    </div>
+  </aside>
+  <main class="main">
+    <div class="content">
+      <article role="main">
+        <label class="toc-overlay-icon toc-content-icon{% if furo_hide_toc %} no-toc{% endif %}" for="__toc">
+          <i class="icon"><svg><use href="#svg-toc"></use></svg></i>
+        </label>
+
+        {% include "partials/download_button.html" %}
+
+        {% block content %}{{ body }}{% endblock %}
```

Comment:
I noticed that you copied this over from [furo/page.html](https://github.com/pradyunsg/furo/blob/51dd416b5a6fae12addbefd3caef36417d329e77/src/furo/theme/furo/page.html).

Just like line `55` in your code, there's a block content for the body:
```html
{% block content %}{{ body }}{% endblock %}
```

I did a diff and it looks like this is the only addition to the file. Would it be possible instead to do something like:

```html
{% extends "page.html" %}

{% block content %}

{% include "partials/download_button.html" %}

{{ body }}

{% endblock %}
```

---

This is just a hunch though. I'm fine with including this for now since the resulting HTML should be the same 😄

Reply:
sure enough!  I even thought about that and cooked up some hair-brained reason why it wasn't going to work here.  but it does!  thanks :)